### PR TITLE
Web support + more generic @bothrs/translations

### PR DESCRIPTION
**Pull request by thgh**
(dit is een voorstel)

## What
* Add web support
* Require explicit use of Airtable data parser by using `useAirtableTranslations` (breaking change)
* Remove `startupLanguage` and use the standard `lng` instead (breaking change)
* Move `dataFormatter` to `fetchOptions.parse` (breaking change)
* Add `useHealthblocksTranslations()`

## How
* Should work in any project after making these replacements:
  * startupLanguage => lng
  * useTranslations => useAirtableTranslations

## Ticket
[Ticket](ticketurl)

### CODE REVIEW

**Comment by TijsM** [comment]
@thgh so if I understand it correctly specific dataformatters are now included in the package? so you're not able to pass a dataformatter yourself? 

wouldn't it make more sense to keep de dataformatter and just expose an the dataformatters?

I think this allows more projects to use this package...

WDYT?

**Comment by thgh** [comment]
You can pass any data formatter using the `fetchOptions.parse` param, like this:
```ts
/** Pass full fetchOptions */
export function GenericExample() {
  const ready = useTranslations({
    lng: 'nl',
    fetchOptions: {
      loadPath: 'https://api.i18next.com/example.json',
      parse: (text) => JSON.parse(text),
    },
  })
  return <div>{ready ? i.t('Ready') : 'Loading'}</div>
}
```

**Comment by TijsM** [comment]
@thgh aaah super! 👍

File: packages/translations/README.md [hunk 1]
```diff
@@ -2,43 +2,41 @@
...
-    fallbackLng: 'nl',
     loadPath: api + 'translations',
-    dataFormatter: data => {
-      return data.translations
-    },
+    // Any other i18next options
+    lng: 'nl',
+    fallbackLng: 'nl',
   })
+  return <div>{translationsLoaded ? i.t('Ready') : 'Loading'}</div>
```
Comment (bramvanhoutte) [inline packages/translations/README.md:28]: Where's the variable `i` coming from?

Also van we use fully written variable names to make it clear what the value of the variable is?

File: packages/translations/src/examples.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import i from 'i18next'
```
Comment (bramvanhoutte) [inline packages/translations/src/examples.tsx:2]: Can you import this as something more readable? E.g. `i18n`

**Review by bramvanhoutte — CHANGES_REQUESTED** [review]

**Review by TijsM — APPROVED** [review]

File: packages/translations/src/examples.tsx (hunk 2 above)
Comment (TijsM) [reply packages/translations/src/examples.tsx]: done

File: packages/translations/README.md (hunk 1 above)
Comment (TijsM) [reply packages/translations/README.md]: done

**Review by bramvanhoutte — APPROVED** [review]

**Pull request merged**